src/test/resources/database/insert-data-ci_repository_test.sql

```diff
@@ -0,0 +1,16 @@
+-- Add test books
```

Comment:
Better divide it into the separate scripts. And insert only the data needed for testing